crates/prover/src/da_block_handler.rs

```diff
@@ -182,6 +182,10 @@ where
                 l1_block.header().height(),
             );
 
+            // Make sure all sequencer commitments are stored in ascending order.
+            // We sort before checking ranges to prevent substraction errors.
+            sequencer_commitments.sort();
+
```

Comment:
```suggestion
```

Reply:
can you add those comments on the new sort?